app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -40,257 +39,206 @@
 
             <!-- Placeholder. Height gets set at runtime based on container size; the initial value is a hack to keep
                  the detail info offscreen until it's placed properly. May be a better way to do this. -->
-            <com.facebook.drawee.view.SimpleDraweeView
-                android:id="@+id/mediaDetailImageView"
+
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/dimen_250"
-                app:actualImageScaleType="none" />
+                android:orientation="vertical"
+                android:background="@android:color/transparent"
+                android:id="@+id/mediaDetailImageViewSpacer"
+                />
 
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:background="?attr/fragmentCategorisationBackground"
-                android:orientation="vertical"
-                android:padding="@dimen/standard_gap">
+                android:background="?attr/mainBackground"
+                android:orientation="vertical">
 
                 <LinearLayout
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:background="?attr/subBackground"
-                    android:orientation="vertical"
-                    android:padding="@dimen/standard_gap">
+                    android:background="@color/primaryDarkColor"
+                    android:orientation="horizontal"
+                    android:padding="@dimen/quarter_standard_height">
+
 
                     <TextView
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
-                        android:paddingBottom="@dimen/tiny_gap"
+
+                        android:layout_width="@dimen/widget_margin"
```

Comment:
style not applied here?

Reply:
You may want a base `Heading` style that your current `Heading` style should inherit. Also maybe consider `Label` instead of `Heading` for a name.